eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -109,11 +109,18 @@ steps:
     msbuildArguments: "/t:AfterBuild /bl:$(Build.StagingDirectory)\\binlog\\03.Localize.binlog"
 
 - task: MSBuild@1
-  displayName: "Build Final NuGet.exe (via ILMerge)"
+  displayName: "Build NuGet.exe Localized"
   inputs:
     solution: "src\\NuGet.Clients\\NuGet.CommandLine\\NuGet.CommandLine.csproj"
     configuration: "$(BuildConfiguration)"
-    msbuildArguments: "/t:ILMergeNuGetExe /p:ExpectedLocalizedArtifactCount=$(LocalizedLanguageCount) /bl:$(Build.StagingDirectory)\\binlog\\04.ILMergeNuGetExe.binlog"
+    msbuildArguments: "/t:Build /bl:$(Build.StagingDirectory)\\binlog\\04.BuildNuGetExe.binlog"
+
+- task: MSBuild@1
```

Comment:
If you look at the logs for `Build NuGet.exe Localized`, you will see that we're now ilmerging twice. 
We need to build twice, but no need to ilmerge twice.

Reply:
Ok, now it builds twice and ILMerge once

04.BuildNuGetExe.binlog

![Build2nd](https://user-images.githubusercontent.com/1192347/191164982-44586257-5dc5-48ea-b69a-01ed0cd84e9b.png)

05.ILMergeNuGetExe.binlog

![image](https://user-images.githubusercontent.com/1192347/191165057-74970eff-b1fa-4216-9528-07039405dcf7.png)